.rubocop.yml

```
@@ -1,2 +1,2 @@
 inherit_from:
-  - https://raw.githubusercontent.com/facebook/chef-cookbooks/master/.rubocop.yml
+  - https://raw.githubusercontent.com/facebook/chef-cookbooks/master/.rubocop_55.yml
```

The default branch is `main`
```suggestion
  - https://raw.githubusercontent.com/facebook/chef-cookbooks/main/.rubocop_55.yml
```